contracts/utils/structs/BitMap.sol

```diff
@@ -0,0 +1,32 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+/**
+ * @dev Library for managing uint256 to bool mapping in a compact and efficient way, providing the keys are sequential.
+ * Largelly inspired by Uniswap's https://github.com/Uniswap/merkle-distributor/blob/master/contracts/MerkleDistributor.sol[merkle-distributor].
+ */
+library BitMap {
+    struct UintBitMap {
```

Comment:
Do we plan to have any non-`Uint` variants of this? Can we just call it `BitMap`?

Reply:
Having a struct with the same name as the library its in causes warning message.

Reply:
Looking at our other libraries, they are usually plural, so we should name the library `BitMaps` and the struct `BitMap`. This avoids the warning message.